src/include/apparmor/aa-config.rb

```diff
@@ -33,20 +33,26 @@ def initialize_apparmor_aa_config(include_target)
       Yast.import "Label"
       Yast.import "Popup"
       Yast.import "Wizard"
+      Yast.import "Service"
     end
 
     def changeAppArmorState(aaEnabled)
       error = nil
       aaAction = ""
+      service = Yast::ServiceClass.new
 
       if aaEnabled == true
-        aaAction = "apparmor:enable"
+        error = service.start("apparmor")
+	if error == true
+           error = service.enable("apparmor")
+	end
       else
-        aaAction = "apparmor:disable"
+        error = service.stop("apparmor")
+        if error == true
+           error = service.disable("apparmor")
+	end
       end
 
-      error = SCR.Execute(path(".aaconf"), aaAction)
-
       if error != nil && Ops.is_string?(error)
```

Comment:
with new logic this cannot happen anymore as error is true/false and never string